### DISCUSSION TITLE
Increase Redis default password length

### DISCUSSION THREAD

**Discussion by andyzukunft** [coollabsio/coolify#1490] [Feature Requests]
Hey,

the Redis password is somewhat small-ish. It might not matter if the access is restricted however I am hosting a Redis and for development purposes I like to access Redis running on the staging machine. This means Redis is accessible over the internet.

Is it possible to double or even triple the default generated password for Redis?

**Comment by andrasbacsai** [comment]
I will increase the default password length in the upcoming version for all databases.
